src/utils/darkModeContext.ts

```diff
@@ -0,0 +1,8 @@
+import { createContext } from "react";
+
+type DarkModeContext = {
+  darkMode: boolean;
+  changeDarkMode: (darkMode: boolean) => void;
+};
+
+export const DarkModeContext = createContext<DarkModeContext>(undefined);
```

Comment:
The type of the context value is `DarkModeContext` which does not accept `undefined` but the initial value is `undefined`? Sounds fishy. :)

Reply:
You're right i'll update this